### ISSUE TITLE
Cannot Use Mod

### ISSUE THREAD

**Issue by UyTheo** [Rabbitminers/Extended-Flywheels#39]
Everytime I install this mod and your other mod, Extended Cogwheels and launch in 1.18.2 the game says the mods fail to load correctly

**Comment by annaleighsmith** [comment]
I am having a similar issue with 1.19.2. Game won't start instant crash

**Comment by xx-Guardian-xx** [comment]
same
![Capture](https://github.com/Rabbitminers/Extended-Flywheels/assets/140301633/1804bbd4-69ca-40fe-b675-e567f5397d0a)

**Comment by Xelbayria** [comment]
Yup, me, too. 

I think Create already included the Flywheel v6.9. I believed the Changelog on Create's for 1.18.2 is somewhat vaguely not clear. 
![flywheel-v6 9](https://github.com/Rabbitminers/Extended-Flywheels/assets/12866666/918a34d5-e9eb-443f-b8c9-5eef9d289380)

and also look at the [changelog](https://www.curseforge.com/minecraft/mc-mods/create/files/4625533) where it's saying **Now using Flywheel v0.6.9**

It seems that Extended Flywheel needed to be updated to detect the latest version of Flywheel. that goes for Cogwheel, too

**Comment by QWXL** [comment]
Maybe because the Extended Flywheel does not support Create 0.5.1
(I dont know why the author delayed updating this mod, it's obvious that it's not that hard!

**Comment by Xelbayria** [comment]
> Maybe because the Extended Flywheel does not support Create 0.5.1 (I dont know why the author delayed updating this mod, it's obvious that it's not that hard!

just a friendly piece of advice:  
if u don't know how to code, then u know it's not a easy work as you think it is because it can take some time to check for any changes. Create included Flywheel from 51C now or so. that's a big change.  normally, you have 2 mods: Create and Flywheel, now you just need 1 mod (Create) which already has Flywheel included. 

and The dev has a life to tend.  The best option is to go back to the previous one where Create & Flywheel are working fine with Extended Flywheel | Cogwheels. This way we can patiently wait for Flywheel to be updated or Extended Flywheel | Cogwheels to be updated.

**Comment by QWXL** [comment]
> 

No, I'm also a Mod developer and a Minecraft server owner, and I understand that any developer has his or her own things to do, but today it's been almost 1 month since the Create 0.5.1 update, and I don't think that anything significant has happened to this author, as his Github is still constantly active.
The vast majority of Create-affiliated mods have been updated, but not Extended Flywheel.
So, I still don't understand why the author is still delaying updating Extended Flywheel, when it's not that hard or he's been using it for a month now.

**Comment by QWXL** [comment]
> > Maybe because the Extended Flywheel does not support Create 0.5.1 (I dont know why the author delayed updating this mod, it's obvious that it's not that hard!
> 
> just a friendly piece of advice: if u don't know how to code, then u know it's not a easy work as you think it is because it can take some time to check for any changes. Create included Flywheel from 51C now or so. that's a big change. normally, you have 2 mods: Create and Flywheel, now you just need 1 mod (Create) which already has Flywheel included.
> 
> and The dev has a life to tend. The best option is to go back to the previous one where Create & Flywheel are working fine with Extended Flywheel | Cogwheels. This way we can patiently wait for Flywheel to be updated or Extended Flywheel | Cogwheels to be updated.

No, I'm also a Mod developer and a Minecraft server owner, and I understand that any developer has his or her own things to do, but today it's been almost 1 month since the Create 0.5.1 update, and I don't think that anything significant has happened to this author, as his Github is still constantly active.
The vast majority of Create-affiliated mods have been updated, but not Extended Flywheel.
So, I still don't understand why the author is still delaying updating Extended Flywheel, when it's not that hard or he's been using it for a month now.

**Comment by Xelbayria** [comment]
> No, I'm also a Mod developer and a Minecraft server owner, and I understand that any developer has his or her own things to do, but today it's been almost 1 month since the Create 0.5.1 update, and I don't think that anything significant has happened to this author, as his Github is still constantly active. The vast majority of Create-affiliated mods have been updated, but not Extended Flywheel. So, I still don't understand why the author is still delaying updating Extended Flywheel, when it's not that hard or he's been using it for a month now.

I had a feeling that you are a programmer. tbh, I noticed that it's been a month. I also noticed that the Extended Flywheel & Cogwheel have somewhat updated to account for version 51. If you would take a look at the `gradle.properties` and check out the current version that the mod is at. but it hasn't been uploaded to CF (Curseforge). It would be nice if someone could contact him to let him know whether he forgot about the part where he needs to upload these mods to CF. 

I am not sure whether the recently updated 2 mods would work with the Create with Flywheel.

**Comment by Rabbitminers** [comment]
> No, I'm also a Mod developer and a Minecraft server owner, and I understand that any developer has his or her own things to do, but today it's been almost 1 month since the Create 0.5.1 update

Clearly you don't, if  you did you would be able to realise how unreasonable "it's been almost a month" is. A month is hardly a grand scope of time, besides I do not owe you my free time. This is a hobby and you're going to have to respect that.

> I still don't understand why the author is still delaying updating Extended Flywheel

I'm not delaying anything. It's ready when it's ready, if you would care to look past the open issues you'll see that my other mods are being updated first, along with content updates. So instead of looking at commits without context and assuming I don't give one, you may care to listen to the many progress and status updates that we make on our discord server of the new features, models and textures we are adding,

**Comment by crystalgem1** [comment]
im sorry to be one of those guys asking for a when, but i really loved this mod and also the config where you can make it so that for the later game you need to use harder materials then wood, so i would also like to know when i can play with it again in the newer versions but no pressure, hope you're doing well
